src/main/java/uk/gov/justice/hmpps/prison/service/PrisonerReleaseAndTransferService.java

```diff
@@ -163,7 +165,7 @@ public void transferOutPrisoner(final String prisonerIdentifier, final RequestTo
         // set previous active movements to false
         setPreviousMovementsToInactive(booking);
 
-        final var toLocation = agencyLocationRepository.findByIdAndTypeAndActiveFlagAndDeactivationDateIsNull(requestToTransferOut.getToLocation(), "INST", ActiveFlag.Y).orElseThrow(EntityNotFoundException.withMessage("No %s agency found", requestToTransferOut.getToLocation()));
+        final var toLocation = agencyLocationRepository.findByIdAndTypeAndActiveFlagAndDeactivationDateIsNull(requestToTransferOut.getToLocation(), agencyLocationTypeRepository.findById(AgencyLocationType.INST).orElseThrow(EntityNotFoundException.withMessage("Not Found")), ActiveFlag.Y).orElseThrow(EntityNotFoundException.withMessage("No %s agency found", requestToTransferOut.getToLocation()));
```

Comment:
maybe extract `agencyLocationTypeRepository.findById(AgencyLocationType.INST).orElseThrow(EntityNotFoundException.withMessage("Not Found"))` to `typeFor(AgencyLocationType.INST)` to reduce code noise

Reply:
Done